Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:bullseye
```

Comment:
Instead of
FROM debian:bullseye

Consider:
FROM python:3.8.13-bullseye

Then remove the apt-get for python/pip

Reply:
Thank you.